src/coreclr/vm/methodtable.h

```diff
@@ -1551,11 +1551,10 @@ class MethodTable
 
     inline WORD GetNumIntroducedInstanceFields();
 
-    // Note: This flag MUST be available even from an unrestored MethodTable - see GcScanRoots in siginfo.cpp.
-    DWORD           ContainsPointers()
+    BOOL           ContainsPointers()
```

Comment:
Nit: You can also make it lower-case `bool` since you have added `!!` to normalize the value. It would be nice if the few bool returning properties that are next to each other in this file followed same style.

Reply:
Agreed. I'm planning on going back soon and trying to normalize this pattern.